src/applications/facility-locator/components/search-results-items/common/LocationOperationStatus.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { OperatingStatus } from '../../../constants';
+
+const LocationOperationStatus = ({ operatingStatus }) => {
+  let infoMsg;
+  let classNameAlert;
+  let iconType;
+  if (operatingStatus.code === OperatingStatus.NOTICE) {
```

Comment:
Can we please add an empty line above each `if` block and above the `return` statement?